index.js

```diff
@@ -1,2 +1,125 @@
 #!/usr/bin/env node
-require('./build/deploy')
+'use strict'
+
+const co = require('co')
+const fs = require('mz/fs')
+const inquirer = require('inquirer')
+const path = require('path')
+const rsyncwrapper = require('rsyncwrapper')
+const yargs = require('yargs')
+
+function rsync(opts) {
+	return new Promise((yes, no) => {
+		rsyncwrapper(opts, (err, stdout, stderr, cmd) => {
+			const ret = { stdout, stderr, cmd }
+			return err ? no(Object.assign(err, ret)) : yes(ret)
+		})
+	})
+}
+
+const interactive = co.wrap(function* (conf) {
+	let destIndex = 0
+	if (conf.destinations.length > 1) {
+		const answers = yield inquirer.prompt([{
+			name: 'destIndex',
+			type: 'list',
+			message: 'Which host do you want to deploy to?',
+			choices: conf.destinations.map((dest, index) => ({ name: dest.name, value: index }))
+		}])
+		destIndex = answers.destIndex
+	}
+
+	const typeArgs = yield inquirer.prompt([ {
+		type: 'list',
+		name: 'type',
+		message: 'How would you like to deploy?',
+		choices: [
+			{ name: 'Normal (--dry-run)', value: { dryRun: true } },
+			{ name: 'Normal', value: [] },
+			{ name: 'Delete (--delete --dry-run)', value: { delete: true, dryRun: true } },
+			{ name: 'Delete (--delete)', value: { delete: true } } ]
+	} ])
+
+	return {
+		destIndex,
+		delete: typeArgs.type.delete || false,
+		dryRun: typeArgs.type.dryRun || false,
+	}
+})
+
+//
+// MAIN
+//
+co(function* () {
+	const pkgPath = path.join(process.cwd(), 'package.json')
+	const args = yargs.usage('$0 [options]')
+		.options({
+			'alias': { alias: 'a', type: 'string', desc: '' },
```

Comment:
Can you fill in the description for "alias" so that it shows up in the help message?  (When the description is empty, `--alias` is hidden from the help message)